app/components/option_item/index.tsx

```diff
@@ -214,7 +214,7 @@ const OptionItem = ({
                 value={selected}
                 trackColor={trackColor}
                 thumbColor={thumbColor}
-                testID={`${testID}.toggled.${selected}.${value}`}
+                testID={`${testID}.toggled.${selected}.button`}
```

Comment:
why this change?

Reply:
When I inpect the TestID in devtool, I was seeing lot of toggle buttons had undefined in the place of value.
example `post_priority_picker_item.requested_ack.toggled.false.undefined`
So, instead of appending value with might be undefined, added `button`